src/views/OrderDetail.vue

```diff
@@ -159,11 +159,11 @@ export default defineComponent({
       }
     },
     getEstimatedDeliveryDate(item: any, label: string){
-      if(this.isPreorderBackorderItem(item)){
-        return item.properties.find((property: any) => property.name === "PROMISE_DATE") ? item.properties.find((property: any) => property.name === "PROMISE_DATE")["PROMISE_DATE"] : "";
-      }
       const product = this.getPreorderItemAvailability(item.sku);
       if(product.label === label){
+        if(this.checkPreorderBackorderItem(item)){
```

Comment:
Handle case when we have property already set, but the product is no longer backorder